packages/postgres-database/src/simcore_postgres_database/migration/versions/53e095260441_add_projects_access_rights_and_.py

```diff
@@ -0,0 +1,111 @@
+"""add projects access rights and ownership to user id
+
+Revision ID: 53e095260441
+Revises: 64614dc0fada
+Create Date: 2020-04-24 06:30:31.689985+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "53e095260441"
+down_revision = "64614dc0fada"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "projects",
+        sa.Column(
+            "access_rights",
+            sa.dialects.postgresql.JSONB(),
+            nullable=False,
+            server_default=sa.text("'{}'::jsonb"),
+        ),
+    )
+    # op.execute("UPDATE projects SET access_rights = {}")
+    # op.alter_column("projects", "access_rights", nullable=False)
+    op.alter_column("projects", "creation_date", server_default=sa.text("now()"))
+    # FIXME: this does not work, alembic has no onupdate
+    op.alter_column(
+        "projects",
+        "last_change_date",
+        server_default=sa.text("now()"),
+        # onupdate=sa.text("now()"),
+    )
+    # define conversion fct
+    set_prj_owner_conversion_fct = sa.DDL(
+        f"""
```

Comment:
TIP: can indent and use https://docs.python.org/3.6/library/textwrap.html to remove prefix spaces